solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -435,11 +439,11 @@ library DKG {
 
     /// @notice Cleans up state after DKG completion.
     /// @dev Should be called after DKG times out or a result is approved.
-    modifier cleanup(Data storage self) {
-        _;
+    function cleanup(Data storage self) internal {
```

Comment:
I decided to make `cleanup` a function instead of modifier, because we need to pay the reward to the block submitter before we clean up (cleanup zeroes the submitter address stored in DKG data and we need that address to pay the reward). After this change `cleanup` is executed in the very end of `approveDkgResult` and `notifyDkgTimeout`.

Reply:
What if we rename the function to `complete`? We already call `dkg.start()` function, so maybe it would make sense to call `dkg.complete()`?

Reply:
It makes sense, done in https://github.com/keep-network/keep-core/pull/2690/commits/aea62227d52367e34dc04941ae806ddc3a42faff.